tmt/convert.py

```diff
@@ -129,10 +142,11 @@ def html_to_markdown(html):
         markdown = ""
     else:
         markdown = md_handler.handle(html).strip()
+    assert isinstance(markdown, str)
```

Comment:
I'd probably go with `cast()` here. Line 142 introduces the variable as a string, `cast(str, md_handler.handle(...)).strip()` would make it explicit that only string is assigned to `markdown`, and it'd be applied at the point where "string but it's not visible" value enters tmt code. `assert` is a solution when it's much harder to introduce the correct type, or when it's too late to annotate.

Reply:
Thank you, it works